src/mnemonic/mod.rs

```diff
@@ -34,25 +36,31 @@ pub fn mnemonic_to_entropy(words: Vec<String>) -> Result<[u8; 32]> {
     let bits = bit_vec.join("");
 
     let divider_index: usize = ((bits.len() as f64 / 33.0) * 32.0).floor() as usize;
-    let (entropy_bits, checksum_bits) = bits.split_at(divider_index);
-    // The mobile wallet does not calculate the checksum bits right so
-    // they always and up being all 0
-    if checksum_bits != "0000" {
-        bail!("invalid checksum");
-    }
+    let (entropy_bits, _checksum_bits) = bits.split_at(divider_index);
+
+    // TODO: Implement checksum checking, while still accounting for bug in the
```

Comment:
I disabled this because valid BIP-39 mnemonics will fail this check and I'm new enough at Rust that I didn't immediately see a great way to verify the checksum (AFAICT, it's  take the sha256 of the entropy_bits and compared those to checksum_bits, but those need to be bits and not vector of "1" and "0")